client/tests/integration/components/land-use-action-test.js

```diff
@@ -0,0 +1,122 @@
+
+    assert.equal(this.package.pasForm.dcpPfzoningspecialpermit, 6);
+
+    await fillIn('[data-test-input="dcpZoningspecialpermitpursuantto"]', 'Section 5B');
+    await triggerEvent('[data-test-input="dcpZoningspecialpermitpursuantto"]', 'keyup');
+
+    assert.equal(this.package.pasForm.dcpZoningspecialpermitpursuantto, 'Section 5B');
+
+    await fillIn('[data-test-input="dcpZoningspecialpermittomodify"]', 'Permit 7A');
+    await triggerEvent('[data-test-input="dcpZoningspecialpermittomodify"]', 'keyup');
+
+    assert.equal(this.package.pasForm.dcpZoningspecialpermittomodify, 'Permit 7A');
+  });
+
+  test('User can delete an action', async function(assert) {
+    const projectPackage = this.server.create('package');
+
+    this.package = await this.owner.lookup('service:store').findRecord('package', projectPackage.id, { include: 'pas-form' });
+
+    // Template block usage:
+    await render(hbs`
+      <LandUseAction @pasForm={{this.package.pasForm}}>
+      </LandUseAction>
+    `);
+
+    // Check that we can delete "Change in CityMap"
+    await selectChoose('[data-test-land-use-action-picker]', 'Change in CityMap');
+
+    assert.dom(this.element).includesText('How many Change in CityMap actions?');
+
+    assert.ok(!this.package.pasForm.dcpPfchangeincitymap);
+
+    await fillIn('[data-test-input="dcpPfchangeincitymap"]', 4);
+
+    await triggerEvent('[data-test-input="dcpPfchangeincitymap"]', 'keyup');
+
+    assert.equal(this.package.pasForm.dcpPfchangeincitymap, 4);
+
+    await click('[data-test-delete-button="Change in CityMap"]');
+
+    assert.dom(this.element).doesNotIncludeText('How many Change in CityMap actions?');
+
+    assert.equal(this.package.pasForm.dcpPfchangeincitymap, 0);
+
+    // Check that user can delete "Zoning Special Permit"
+    await selectChoose('[data-test-land-use-action-picker]', 'Zoning Special Permit');
+
+    await fillIn('[data-test-input="dcpPfzoningspecialpermit"]', 6);
+    await triggerEvent('[data-test-input="dcpPfzoningspecialpermit"]', 'keyup');
+
+    assert.equal(this.package.pasForm.dcpPfzoningspecialpermit, 6);
+
+    await fillIn('[data-test-input="dcpZoningspecialpermitpursuantto"]', 'Section 5B');
+    await triggerEvent('[data-test-input="dcpZoningspecialpermitpursuantto"]', 'keyup');
+
+    assert.equal(this.package.pasForm.dcpZoningspecialpermitpursuantto, 'Section 5B');
+
+    await fillIn('[data-test-input="dcpZoningspecialpermittomodify"]', 'Permit 7A');
+    await triggerEvent('[data-test-input="dcpZoningspecialpermittomodify"]', 'keyup');
+
+    assert.equal(this.package.pasForm.dcpZoningspecialpermittomodify, 'Permit 7A');
+
+    await click('[data-test-delete-button="Zoning Special Permit"]');
+
+    assert.equal(this.package.pasForm.dcpPfzoningspecialpermit, 0);
+    assert.equal(this.package.pasForm.dcpZoningspecialpermitpursuantto, '');
+    assert.equal(this.package.pasForm.dcpZoningspecialpermittomodify, '');
+  });
```

Comment:
This deletion test is probably a little too long — it seems like we want to prove that we can delete stuff, but are there really this many clicks involved to set it up for asserting for deletion?

Reply:
I can definitely make it shorter